test/tests.py

```diff
@@ -200,3 +221,30 @@ def test_0006_invalidIncorrectUri(self):
         incorrectUri = "/_cluster/incorrectUri"
         response = requests.get(f'{self.proxy_endpoint}{incorrectUri}', auth=self.auth, verify=False)
         self.assertEqual(response.status_code, HTTPStatus.METHOD_NOT_ALLOWED)
+
+    def test_0007_OSB(self):
+        cmd = "docker ps | grep 'migrations/migration_console:latest' | awk '{print $NF}'"
+        container_name = subprocess.getoutput(cmd).strip()
+
+        if container_name:
+            cmd_exec = f"docker exec {container_name} ./runTestBenchmarks.sh"
+            logger.warning(f"Running command: {cmd_exec}")
+            subprocess.run(cmd_exec, shell=True)
+        else:
+            logger.error("Migration-console container was not found, please double check that deployment was a success")
+            self.assert_(False)
+
+        source_indices = get_indices(self.source_endpoint, self.auth)
+        target_indices = get_indices(self.target_endpoint, self.auth)
+
+        for index in set(source_indices) & set(target_indices):
```

Comment:
This test provides value as it but I can see this as a future improvement. I'm going to mark this one approved so that we can start seeing the results of this test. I'd like to get this in before I bump the version of some of our dependencies.